nix/nix-prefetch-github.nix

```diff
@@ -7,6 +7,11 @@ in buildPythonPackage {
   outputs = [ "out" "doc" "man" ];
   src = ../.;
   nativeBuildInputs = [ sphinxHook sphinx-argparse ];
+  postInstall = ''
+    for f in $out/bin/* ; do
+        wrapProgram "$f" --prefix PATH : "${git}/bin"
+    done
+  '';
   checkInputs = [ git which pytestCheckHook ];
```

Comment:
technically I think it means you can remove git from the checkInputs? I didn't check.

Reply:
I just checked, and the answer is no. The checks are unit tests, not integration tests on the final installed binary.